### DISCUSSION TITLE
new file swf after maintenance 1330-Release_uther

### DISCUSSION THREAD

**Discussion by Cdrwbw** [fedorovvl/tso_client#320] [General]
[client.zip](https://github.com/fedorovvl/tso_client/files/12999505/client.zip)
new swf for client all servers

**Comment by bellejt** [comment]
thx but not working on independant client on dutch server here

**Comment by Cdrwbw** [comment]
when you change the swf put --skip in command line 
But a t this time maintance is not finish on dutch server

**Reply by bellejt** [reply]
oke but now working in unity and web but still not independant witch updates normaly automatically .Even with --skip

**Comment by Cdrwbw** [comment]
yes but it's if you don't wait

**Reply by bellejt** [reply]
will play on unity for the moment.thx

**Comment by Stribrnak1** [comment]
thx but not working on independant client on czech server here

**Reply by bellejt** [reply]
new SWF is in town
